pipelines/optimus/Loom_schema.md

```diff
@@ -1,44 +1,53 @@
 # What's in the Optimus Pipeline Loom File?
 
-The Loom file is an HDF5 file generated using [Loompy v.2.0.17](http://loompy.org/). It contains UMI-corrected counts as well as multiple metrics for both individual cells (the columns of the matrix; Table 1) and individual genes (the rows of the matrix; Table 2). The tables below document these metrics, list which tools generate them, and define them. This Loom file is an optional output of the Optimus pipeline. The default matrix output of the Optimus pipeline is a Zarr Array. The Loom file is directly derived from the Zarr and contains the same information with only minor header updates for schema compatibility. 
+The Loom file is an HDF5 file generated using [Loompy v.2.0.17](http://loompy.org/). It contains global attributes detailing how counts were generated for the single-cell or single-nuclei parameters ([Table 1](#table-1-global-attributes)). It additionally contains UMI-corrected counts as well as multiple metrics for both individual cells (the columns of the matrix; [Table 2](#table-2-column-attributes-cell-metrics)) and individual genes (the rows of the matrix; [Table 3](#table-3-row-attributes-gene-metrics)). The tables below document these metrics, list which tools generate them, and define them. This Loom file is an optional output of the Optimus pipeline. The default matrix output of the Optimus pipeline is a Zarr Array. The Loom file is directly derived from the Zarr and contains the same information with only minor header updates for schema compatibility. 
 
 **Note**: Loom files generated by Optimus are different from the final Loom file distributed on the [Human Cell Atlas Data Portal](https://data.humancellatlas.org/explore/projects), which removes some of the metadata detailed in this document and contains additional metadata relating to each individual project. 
 
-## Table 1. Column Attributes (Cell Metrics) 
+## Table 1. Global Attributes
+The global attributes in the Loom apply to the whole file, not any specific part. There are two global attributes for the Optimus Loom. 
+
+| Attribute | Details |
+| :-- | :-- |
+| LOOM_SPEC_VERSION | String with the loom file spec version |
+| expression_data_type | String indicating whether counts are generated using exonic only or exonic and intronic reads. The value will be either "exonic" for the sc_RNA parameter or "whole_transcript" for the sn_RNA parameter |
```

Comment:
"indicating whether counts include intronic reads or only exonic reads" ??

The value will be either "exonic" if the pipeline has been run counting_mode set to "sc_rna" or "whole_transcript" if the pipeline has been run with counting_mode set to "sn_rna"

Reply:
Updated!